### PR TITLE
add clusterclaim in concept

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: content/en/concepts/clusterclaim.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+`ClusterClaim` is a cluster-scoped API created by users on a managed cluster, 
+and will be collected and saved into the status of the corresponding `ManagedCluster` on the hub.
+
+`ClusterCaim` is usually used to perform some extensible properties of the managed cluster like 
+clusterID, the version, vendor and cloud provider etc.
+
+## Example 
+
+Here is a `ClusterClaim` example.
```
Comment (dhaiducek) [inline content/en/concepts/clusterclaim.md:22]:
```suggestion
Here is a `ClusterClaim` example specifying a clusterID:
```

File: content/en/concepts/clusterclaim.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+apiVersion: cluster.open-cluster-management.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: id.open-cluster-management.io
+spec:
+  value: 95f91f25-d7a2-4fc3-9237-2ef633d8451c
+```
+
+After applying the `ClusterClaim` above to any managed cluster, the value of the `ClusterClaim` 
+will be instantly reflected in the `ManagedCluster` on the hub cluster.
```
Comment (dhaiducek) [inline content/en/concepts/clusterclaim.md:33]:
```suggestion
is reflected in the `ManagedCluster` on the hub cluster:
```

File: content/en/concepts/clusterclaim.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+<!-- spellchecker-enable -->
+
+## What is ClusterClaim?
+
+`ClusterClaim` is a cluster-scoped API created by users on a managed cluster, 
+and will be collected and saved into the status of the corresponding `ManagedCluster` on the hub.
+
+`ClusterCaim` is usually used to perform some extensible properties of the managed cluster like 
+clusterID, the version, vendor and cloud provider etc.
```
[the remark this answers is not in the document]
Comment (zhiweiyin318) [reply content/en/concepts/clusterclaim.md]: done.

File: content/en/concepts/clusterclaim.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+<!-- spellchecker-disable -->
+
+{{< toc >}}
+
+<!-- spellchecker-enable -->
+
+## What is ClusterClaim?
+
+`ClusterClaim` is a cluster-scoped API created by users on a managed cluster, 
+and will be collected and saved into the status of the corresponding `ManagedCluster` on the hub.
```
[the remark this answers is not in the document]
Comment (zhiweiyin318) [reply content/en/concepts/clusterclaim.md]: done

File: content/en/concepts/clusterclaim.md (hunk 1 above)
Comment (zhiweiyin318) [reply content/en/concepts/clusterclaim.md]: done

File: content/en/concepts/clusterclaim.md (hunk 2 above)
Comment (zhiweiyin318) [reply content/en/concepts/clusterclaim.md]: done

File: content/en/concepts/clusterclaim.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (zhiweiyin318) [reply content/en/concepts/clusterclaim.md]: thanks for the wording suggestions @dhaiducek

**Comment by dhaiducek** [comment]
/lgtm

I don't have as much technical knowledge about ClusterClaims, so I'll leave final approval to @qiujian16 🙂

File: content/en/concepts/clusterclaim.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+<!-- spellchecker-disable -->
+
+{{< toc >}}
+
+<!-- spellchecker-enable -->
+
+## What is ClusterClaim?
+
+`ClusterClaim` is a cluster-scoped API available to users on a managed cluster.
```
Comment (qiujian16) [inline content/en/concepts/clusterclaim.md:14]: there are some reserved cluster name we need to mention

Comment (zhiweiyin318) [reply content/en/concepts/clusterclaim.md:14]: done. introduced the reserved clusterclaims in the doc.

File: content/en/concepts/clusterclaim.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+the status of the corresponding `ManagedCluster` object on the hub.
+
+## Usage
+
+`ClusterCaim` is used to specify additional properties of the managed cluster like 
+the clusterID, version, vendor and cloud provider. We defined some reserved `ClusterClaim` :
+
+* `id.k8s.io`: A unique identifier for the managed cluster.
+* `kubeversion.open-cluster-management.io`: The kubernetes version of the managed cluster.
+* `platform.open-cluster-management.io`: The platform where the managed cluster is running on, like AWS, GCE.
```
Comment (qiujian16) [inline content/en/concepts/clusterclaim.md:25]: hrm maybe just id.k8s.io...

Comment (zhiweiyin318) [reply content/en/concepts/clusterclaim.md]: ok. fixed.

**Pull request merged**